src/containers/Tenant/Query/i18n/ru.json

```diff
@@ -15,5 +15,10 @@
 
   "preview.title": "Предпросмотр",
   "preview.not-available": "Предпросмотр недоступен",
-  "preview.close": "Закрыть предпросмотр"
+  "preview.close": "Закрыть предпросмотр",
+
+  "method-description.script": "YQL Script – для скриптов, совмещающих DDL и DML-конструкции. API call: schema.scripting",
+  "method-description.scan": "Scan – только читающие запросы, потенциально читающие много данных. API call: table.ExecuteScan",
+  "method-description.data": "Data – DML-запросы для изменения и выброки данных в режиме изоляции Serializable. API call: table.executeDataQuery",
+  "method-description.query": "YQL – QueryService – любые запросы. Экспериментальная перспективный метод, которые заменит все остальные. API call: query.ExecuteScript"
```

Comment:
"Экспериментальная", "которые" - wrong ending

"Заменит" - I think it's better to mention, that this will be in the future. Currently it is not always a valid alternative to all other query modes